libraries/cyclestreets-view/src/main/res/layout/texthistory.xml

```diff
@@ -3,6 +3,6 @@
   android:id="@android:id/text1"
   android:layout_width="fill_parent"
   android:layout_height="?android:listPreferredItemHeight"
-  android:textAppearance="?android:textAppearanceMediumInverse"
+  android:textAppearance="?android:textAppearanceMedium"
```

Comment:
@HilaryN  how is this change related?  I assume this changes the colour in some way but I'm curious because I don't see any mention of that in the issue description or the PR header comment.

Reply:
Apologies for not making that clear.
It's a separate bug fix which I tagged on here as it's tiny.
It shows up when you go through the steps Si mentions in #93. In the suggestions dropdown, the items in the list are invisible as they are white text on a white background.  They only appear when you tap on the list and the text/background change colour.
In this set of screenshots the first one is typing the route number in first time. I then accept it.
In the second, you can see (with difficulty) that as I type, a white dropdown has appeared (with one item).  But it's white on white so you can't see the actual item.  The last screenshot is when I touch the item and the background colour changes to green:

<img width="719" alt="Open route dropdown 3" src="https://user-images.githubusercontent.com/59799824/91456779-93696200-e87b-11ea-82fc-051589f734d1.PNG">

This set of screenshots shows the dropdown with 2 items in the list- the second and third show the list with the first and second items selected:

<img width="948" alt="Open route dropdown 4" src="https://user-images.githubusercontent.com/59799824/91457341-273b2e00-e87c-11ea-88fc-a7423c56473f.PNG">

Maybe it's an issue which appeared subsequent to the other problem being logged but as it was impeding me doing testing I thought I'd fix it at the same time.